pandas/tests/indexes/timedeltas/test_tools.py

```diff
@@ -57,6 +57,17 @@ def test_to_timedelta(self):
         expected = TimedeltaIndex([np.timedelta64(1, "D")] * 5)
         tm.assert_index_equal(result, expected)
 
+    def test_to_timedelta_dataframe(self):
+        # GH 11776
+        arr = np.arange(10).reshape(2, 5)
+        df = pd.DataFrame(np.arange(10).reshape(2, 5))
+        for arg in (arr, df):
+            with pytest.raises(TypeError, match="1-d array"):
+                to_timedelta(arg)
+            for errors in ["ignore", "raise", "coerce"]:
+                with pytest.raises(TypeError, match="1-d array"):
```

Comment:
shouldn't this actually have different behavor for coerce / ignore? e.g. this should *not* raise.

Reply:
with a 2D input it always raises

Reply:
ahh ok, its to_timedelta, nvm